docs/index.bs

```diff
@@ -2094,7 +2094,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
                 1. Invoke [=Batch Cache Operations=] with |operations|. If this [=throws=] an exception, set |errorData| to the exception.
                 1. [=Queue a task=], on |cacheJobPromise|'s [=relevant settings object=]'s [=responsible event loop=] using the [=DOM manipulation task source=], to perform the following substeps:
                     1. If |errorData| is null, resolve |cacheJobPromise| with undefined.
-                    1. Else, reject |cacheJobPromise| with a [=exception/create|new=] [=exception=] with |errorData| and a user agent-defined [=exception/message=], in |realm|.
+                    1. Else, reject |cacheJobPromise| with a [=exception/create|new=] [=exception=] with |errorData| and a user agent-defined [=DOMException/message=], in |realm|.
```

Comment:
What you have done here is OK but the default is a user agent-defined message, so you can instead just omit " and a user agent-defined message".

Reply:
done.